### DISCUSSION TITLE
Ispindel freezes after deep sleep

### DISCUSSION THREAD

**Discussion by itaiplt** [universam1/iSpindel#615] [Q&A]
Hello, 
I have assembled and flashed 3 ispindel based on cherry philip board.
All boards have the same behavior which leads me to belive I am having compatibility\hardware\firmware issues. 
Before I am buying a new batch pf wemos boards I wanted to check if theres somting that I can do. 

background: 
1. Firmware 7.3.3 
2. Verified all soldering resistors and diode. 
3. When in AP mode all the comunication works as it should (saves setting, no disconnections, continuous measurements) 
4. Comunication to Wifi network is OK, I am getting 1 measurments before the unit goes into deep sleep forever. 
5. When I am using the reset button the device wakes, send a measurement and stalls in deep sleep again. 
6. Checked the BAT43 diode direction. 
7. Tried flashing diffrant firmwares 
8.I am adding the pictures of the PCBA maybe i missed something.

![1](https://github.com/universam1/iSpindel/assets/156169469/471f6975-41d0-440e-b724-d99420e3d5e0)
![2](https://github.com/universam1/iSpindel/assets/156169469/9b1c4553-9b57-4973-842f-0e187000f2b2)

Logs (I used 2 serial speeds to get all the lines and not gibrish):
> ets Jan  8 2013,rst cause:2, boot mode:(3,6)
> 
> load 0x4010f000, len 3424, room 16
> tail 0
> chksum 0x2e
> load 0x3fff20b8, len 40, room 8
> tail 0
> chksum 0x2b
> csum 0x2b
> v00090d70
> ~ld
> rf cal sector: 1020
> freq trace enable 0
>  ets Jan  8 2013,rst cause:2, boot mode:(3,6)
>  FW 7.3.3
> 2.2.2-dev(38a443e)
> Worker run!
> mounting FS... mounted!
> reading config file
> parsed config:
> {"Name":"1","Token":"20ano7pmlblpka6a3rg1ns4gh2","Sleep":30,"Server":"brew-spy.c                                                                             om","API":3,"Port":80,"Channel":0,"URI":"/api/ispindel","Username":"","Password"                                                                             :"","Job":"ispindel","Instance":"000","Hassio":false,"UseHTTPS":false,"Vfact":19                                                                             1.8000031,"TS":0,"OWpin":12,"POLY":"-0.00031*tilt^2+0.557*tilt-14.054","SSID":"T                                                                             est WiFi","PSK":"12345678","Offset":[0,0,0,0,0,0]}
> offsets not available
> Boot-Mode: Deep-Sleep Wake
> 
> woken from deepsleep, normal mode
> Samples:42 min:59.45 max:88.98 time:756
> x: -168 y: 15272 z: -5700
> Tilt: 74.65
> Tacc: 28.34
> Volt: 4.84
> Temp: 27.44
> Gravity: 25.80
> After waiting 1201ms, result 3
> IP: 192.168.3.128
> 
> calling HTTP
> {"name":"1","ID":2050561,"token":"20ano7pmlblpka6a3rg1ns4gh2","angle":74.6511764                                                                             5,"temperature":27.4375,"temp_units":"C","battery":4.843586922,"gravity":25.7991                                                                             3712,"interval":30,"RSSI":-61}HTTPAPI: posting
> code: 200
> {
>    "status" : 200,
>                     "ts" : "2024-01-10 06:47:58",
>                                                    "data" : { }
>                                                                }
> 
> Final-sleep: 30s; RT: 2572
>  ets Jan  8 2013,rst cause:2, boot mode:(3,6)

Thanks!

**Comment by pppedrillo** [comment]
You need to use 4.7K resistor
In your picture you have soldered 560 ohm

**Comment by itaiplt** [comment]
The colours seem a bit strange in the pic, but it is 4.7K resistor, Also verified with measurement.
In the meantime I replaced the wemos chip with another batch and the units works fine.
So I do suspect that the batch is just bad.

thanks

**Reply by pppedrillo** [reply]
Right, there were many issues with dodgy wemos modules. Great you have problem sorted. 
Happy brewing!

**Reply by thegreatgunbantoad** [reply]
Yeah this is the typical behaviour of a bad Wemos clone.  When I started building these all you could get was 3rd party copies and they sucked, 1 in about 5 was just useless.
